censusdis/datasets.py

```diff
@@ -512,75 +511,75 @@
 
 
 DATASET_REFERENCE_URLS = { 
-	ABS_CB: "http://api.census.gov/data/2017/abscb",
-	ABS_CBO: "http://api.census.gov/data/2017/abscbo",
-	ABS_CS: "http://api.census.gov/data/2017/abscs",
+	ABS_CB: "http://api.census.gov/data/2020/abscb",
```

Comment:
It looks like a side effect of switching from `index` to `zip` was that we now end up with the most recent year rather than the oldest year URL. This is a good thing.